### ISSUE TITLE
detect collisions in SvgImageTranslatorBase.CalculateVariableName

### ISSUE THREAD

**Issue by dittodhole** [dittodhole/dotnet-Svg.Contrib.Render#9]

**Comment by dittodhole** [comment]
Meanwhile, I'd like to share a workaround:

```csharp
public class CustomSvgImageTranslator : Svg.Contrib.Render.XXX.SvgImageTranslator
{
    protected override string CalculateVariableName(string imageIdentifier)
    {
      imageIdentifier = new string(imageIdentifier.Reverse().ToArray());
      var variableName = base.CalculateVariableName(imageIdentifier);
      return variableName;
    }
}
```

**Comment by dittodhole** [comment]
fixed with v0.21
